javascript/packages/orchestrator/src/spawner.ts

```diff
@@ -0,0 +1,249 @@
+        "{{PORT}}",
+        PROMETHEUS_PORT.toString(),
+      ),
+      nodeMultiAddress,
+      opts.userDefinedTypes,
+    );
+  } else {
+    const nodeIdentifier = `${podDef.kind}/${podDef.metadata.name}`;
+    const fwdPort = await client.startPortForwarding(
+      endpointPort,
+      nodeIdentifier,
+    );
+    const nodePrometheusPort = await client.startPortForwarding(
+      PROMETHEUS_PORT,
+      nodeIdentifier,
+    );
+
+    const listeningIp = opts.local_ip || LOCALHOST;
+
+    networkNode = new NetworkNode(
+      node.name,
+      WS_URI_PATTERN.replace("{{IP}}", listeningIp).replace(
+        "{{PORT}}",
+        fwdPort.toString(),
+      ),
+      METRICS_URI_PATTERN.replace("{{IP}}", listeningIp).replace(
+        "{{PORT}}",
+        nodePrometheusPort.toString(),
+      ),
+      nodeMultiAddress,
+      opts.userDefinedTypes,
+    );
+  }
+
+  networkNode.group = node.group;
+
+  if (parachain) {
+    const paraId = parachain.id;
+    if (!network.paras[paraId])
+      network.addPara(
+        paraId,
+        parachain.chainSpecPath,
+        parachain.wasmPath,
+        parachain.statePath,
+      );
+    networkNode.parachainId = paraId;
+    networkNode.para = parachain.para;
+    network.addNode(networkNode, Scope.PARA);
+  } else {
+    network.addNode(networkNode, Scope.RELAY);
+  }
+
+  // Display info about the current node
+  let logTable = new CreateLogTable({
+    colWidths: [20, 100],
+    doubleBorder: true,
+  });
+  logTable.pushTo([
+    ["Pod", decorators.green(node.name)],
+    ["Status", decorators.green("Running")],
+  ]);
+  if (node.overrides && node.overrides.length > 0) {
+    logTable.pushTo([
+      [
+        {
+          colSpan: 2,
+          content: `with ${decorators.yellow("Overrides")}...`,
+        },
+      ],
+    ]);
+
+    for (const override of node.overrides) {
+      logTable.pushTo([
+        ["local_path", override.local_path],
+        ["remote name", override.remote_name],
+      ]);
+    }
+  }
+  if (opts.monitorIsAvailable) {
+    const loki_url = getLokiUrl(
+      namespace,
+      podDef.metadata.name,
+      network.networkStartTime!,
+    );
+    logTable.pushTo([
+      [decorators.green("Grafana logs url"), decorators.magenta(loki_url)],
+    ]);
+  } else {
+    logTable.pushTo([
+      [
+        {
+          colSpan: 2,
+          content: decorators.magenta(
+            "You can follow the logs of the node by running this command: ",
+          ),
+        },
+      ],
+    ]);
+    let logCommand;
+    switch (client.providerName) {
+      case "podman":
+        logCommand = `podman logs -f ${podDef.metadata.name}_pod-${podDef.metadata.name}`;
+        break;
+      case "kubernetes":
+        logCommand = `kubectl logs -f ${podDef.metadata.name} -c ${podDef.metadata.name} -n ${namespace}`;
+        break;
+      case "native":
+        logCommand = `tail -f  ${client.tmpDir}/${podDef.metadata.name}.log`;
+        break;
```

Comment:
Could probably be abstracted per provider

Reply:
Done, thanks!